lib/cylc/network/https/base_client.py

```diff
@@ -22,9 +22,6 @@
 from uuid import uuid4
 import warnings
 
-# Ignore incorrect SSL certificate warning from urllib3 via requests.
-warnings.filterwarnings("ignore", "Certificate has no `subjectAltName`")
```

Comment:
The `subjectAltName` field is defined in our certificate generation logic, so I don't know why we'll have this warning in the first place.